cmake/3rdParty.cmake

```diff
@@ -6,6 +6,8 @@
 #
 #
 
+set(LY_RADEON_GPU_ANALYZER_ENABLED FALSE CACHE BOOL "Whether to download Radeon GPU Analyzer from Github.")
```

Comment:
Can use `O3DE_RADEON_GPU_ANALYZER_ENABLED` for the name of the Cache Variable?
The `LY_*` prefix is a holdover from Lumberyard and going forward we are wanting to use O3DE specific names.